handler/src/main/java/io/netty/handler/ssl/OpenSslCertificateException.java

```diff
@@ -24,6 +27,286 @@
  * SSL Record. This only work when {@link SslProvider#OPENSSL} is used.
  */
 public final class OpenSslCertificateException extends CertificateException {
+    public static final int X509_V_OK;
```

Comment:
Lets delay this change until we can do it in one PR (remove old internal APIs and recommend this new API).